pageserver/src/config.rs

```diff
@@ -104,6 +107,8 @@ pub mod defaults {
 #image_creation_threshold = {DEFAULT_IMAGE_CREATION_THRESHOLD}
 #pitr_interval = '{DEFAULT_PITR_INTERVAL}'
 
+#min_resident_size_override = .. # in bytes
```

Comment:
I would have preferred this to be part of the global `disk_usage_based_eviction` config item.
But, it would require some acrobatics in the config parsing code, which I want to avoid.

Ideas for a better name to make it clear that this relates to disk_usage_based_eviction?

Reply:
What does this setting do?

Reply:
> What does this setting do?

Should be clear from the improved module-level comment, which I pushed slightly after your comment above:
https://github.com/neondatabase/neon/pull/3809/commits/0462563d318e2535a57f9219671e303237869aba

Reply:
@koivunej  thoughts?

Reply:
Well, I of course think it's clear.

But I do sense a general issue how we do not document these in the template (or generate the template automatically using the default properties -- requires a bit macro magic but doable).

I guess it could be more clear in the sense "when should you use it", but so are all of our configs. Like can you use it to resolve an outage, probably not, should you set it for certain customers, probably yes.